docs/samples/explanation/aix/Fetch_20newsgroups/train_rf.py

```diff
@@ -0,0 +1,60 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import pickle
+import sklearn
+
+from sklearn.pipeline import Pipeline
+from sklearn.ensemble import RandomForestClassifier
+from sklearn import datasets
+
+
+class PipeStep(object):
+    """
+    Wrapper for turning functions into pipeline transforms (no-fitting)
+    """
+    def __init__(self, step_func):
+        self._step_func = step_func
+
+    def fit(self, *args):
+        return self
+
+    def transform(self, X):
+        return self._step_func(X)
+
+
+path_to_save = sys.argv[1]
+
+dataset = sys.argv[2]
```

Comment:
Do we need this line?

Reply:
@yuzisun The line was used for testing other dataset
we forgot to remove that, we will fix that later

Reply:
@yuzisun that line has been deleted